docs/CHANGELOG.asciidoc

```diff
@@ -72,6 +72,431 @@ Review important information about the {kib} 7.16.x releases.
+Lens & Visualizations::
+* Adds reference lines to *Lens* {kibana-pull}108342[#108342]
+* Support custom field format in *TSVB* {kibana-pull}101245[#101245]
+Observability::
+* Adds "Add to case" button {kibana-pull}112463[#112463]
+Machine Learning::
+* Adds the ability to change data view in advanced job wizard {kibana-pull}115191[#115191]
+* Adds KQL filter bar, filtering functionality, and compact design to Index data visualizer {kibana-pull}112870[#112870]
+Management::
+* Adds alerting rules management to Transform UI {kibana-pull}115363[#115363]
+* Transforms health alerting rule type {kibana-pull}112277[#112277]
+* Allow fields that start with an underscore in the field list {kibana-pull}111238[#111238]
+Platform::
+* Inspect flyout should be available in editor mode {kibana-pull}109656[#109656]
+Sharing::
+* Short URL service now supports human readable slugs for short URLs {kibana-pull}107859[#107859]
+
+For more information about the features introduced in 7.16.0, refer to <<whats-new,What's new in 7.16>>.
+
+[[enhancements-and-bug-fixes-v7.16.0]]
+=== Enhancements and bug fixes
+
+For detailed information about the 7.16.0 release, review the enhancements and bug fixes.
+
+[float]
+[[enhancement-v7.16.0]]
+=== Enhancements
+Alerting::
+* The Alerting event log indices, named `kibana-event-log-*`, are set to hidden indices via the `index.hidden: true` index setting {kibana-pull}117470[#117470]
+* Removes TLS requirement for alerting when security is enabled {kibana-pull}115234[#115234]
+* Shows execution duration on Rule Details view {kibana-pull}114719[#114719]
+* Shows last execution duration on Rule Management view {kibana-pull}113935[#113935]
+* Allow rule types to specify a default and minimum interval {kibana-pull}113650[#113650]
+APM::
+* Adds advanced mode toggle for the APM failed transactions table {kibana-pull}114363[#114363]
+* Adds context popover for APM latency correlations & failed transactions correlations {kibana-pull}113679[#113679]
+* Adds chart for failed transactions correlations tab {kibana-pull}110172[#110172]
+* Correlations chart enhancements {kibana-pull}113039[#113039]
+Canvas::
+* Adds Tag Cloud support {kibana-pull}106858[#106858]
+Dashboard::
+* Dashboard PDF and PNG reports can now be generated without requiring users to first save {kibana-pull}108553[#108553]
+Discover::
+* Remember most recent "hide chart" setting {kibana-pull}114534[#114534]
+* Adds "Chart options" menu {kibana-pull}112453[#112453]
+* Adds "Open in Discover" to dashboard panels {kibana-pull}112396[#112396]
+Elastic Security::
+For the Elastic Security 7.16.0 release information, refer to {security-guide}/release-notes.html[_Elastic Security Solution Release Notes_].
+Fleet::
+* Adds link to integration data retention documentation {kibana-pull}115353[#115353]
+* Adds agent modal {kibana-pull}114830[#114830]
```

Comment:
I'm on the fence about whether to include this one because I think it falls into the category of things users will see when they interact with the UI. If we keep it, I think we need to be more descriptive and say something like:

Adds prompt for users to add an agent if they add an integration to an agent policy with no agents

That's a mouthful, though. WDYT?

Reply:
Better than what's there! Updated.